sklearn/neighbors/binary_tree.pxi

```diff
@@ -293,11 +293,11 @@ Examples
 Query for k-nearest neighbors
 
     >>> import numpy as np
-
+    >>> from sklearn.neighbors import KDTree
```

Comment:
Hi, I am not an expert but looking at the source code and documentation for other ``neighbours``, for eg. [Ball tree](http://scikit-learn.org/stable/modules/generated/sklearn.neighbors.BallTree.html), this would be incorrectly importing ``KDTree`` in the docs. I suppose we can also use the ``{BinaryTree}`` directive used here. Also, it would probably be better to add the import statement for the below examples too. Thanks!